### PR TITLE
Fix #1: Add <noscript>

### LINKED ISSUE #1: Add <noscript>
Add a `<noscript>` tag indicating that JS is necessary for the website to successfully work.

### PR DESCRIPTION
**Pull request by sdulal**

### CODE REVIEW

File: questions/templates/questions/home.html [hunk 1]
```diff
@@ -27,5 +27,6 @@
   <body>
     <div class="container" id="content"></div>
     <script src="{% static 'questions/home/app.js' %}" type="text/babel"></script>
+    <noscript>Want to use this website? Turn on JavaScript!</noscript>
```
Comment (kevinlee12) [inline questions/templates/questions/home.html]: Might be nice if we include a link on how to turn on Javascript, here's one: http://www.enable-javascript.com/

**Review by kevinlee12 — CHANGES_REQUESTED** [review]
Once the link is placed in, I'm happy to merge. Thanks!

**Comment by sdulal** [comment]
Added the link!

**Review by kevinlee12 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**